### PR TITLE
Change default data amount to 1200 points- about 3 years.

### PR DESCRIPTION
**Pull request by bjedrzejewski**
I think we should merge this first, to close the issue, rather than be dependent on bitflux adding features. This can be customized later as a separate feature.

Performance is good (can't see a difference).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
     "angular-animate": "^1.4.8",
     "angular-resource": "^1.4.8",
     "d3fc": "^5.2.0",
-    "BitFlux": "ScottLogic/BitFlux#cf81eab098cf6c6f4a4bd36269d9cfe21d4b449b",
+    "BitFlux": "bjedrzejewski/BitFlux#dddca873ab952c7d9f792c54155b4b73594e7ab3",
```
Comment (owennw) [inline package.json]: Which version of BitFlux is this repo updated to?

**Comment by owennw** [comment]
Would it be difficult for you to make a change to BitFlux to add this to their API and PR that in? @jleft

File: package.json (hunk 1 above)
Comment (bjedrzejewski) [inline package.json]: Latest dev

**Comment by bjedrzejewski** [comment]
I think expanding API would be ideal. This could go together with the API for setting and getting indicators.

File: package.json (hunk 1 above)
Comment (owennw) [inline package.json]: The current BitFlux we're using is https://github.com/ScottLogic/BitFlux/commit/cf81eab098cf6c6f4a4bd36269d9cfe21d4b449b, so your change will actually be updated the version of BitFlux too. Do we want that?

Comment (bjedrzejewski) [inline package.json]: @owennw I would say say, but I don't have a strong preference.

Comment (akhmorgan) [inline package.json]: why is this now pointing to a forked version rather than the main bitflux repo?

Comment (akhmorgan) [inline package.json]: ah never-mind. Got it.

**Comment by jleft** [comment]
We have an open PR to add this to the BitFlux API, it's a fairly straightforward change to make.

https://github.com/ScottLogic/BitFlux/pull/660

**Comment by owennw** [comment]
Given the PR for BitFlux has now been merged, and this PR changes our version to use the latest dev anyway, can you re-work this PR to use the BitFlux repo and access the new API method?

**Comment by bjedrzejewski** [comment]
This will be reworked using new api

**Pull request closed without merging**